pkg/beholder/config.go

```diff
@@ -16,7 +16,10 @@ type Config struct {
 	// OTel Resource
 	ResourceAttributes []otelattr.KeyValue
 	// Message Emitter
-	EmitterExportTimeout time.Duration
+	EmitterExportTimeout      time.Duration
+	EmitterExportInterval     time.Duration
+	EmitterExportMaxBatchSize int
+	EmitterMaxQueueSize       int
 	// Batch processing is enabled by default
 	// Disable it only for testing
 	EmitterBatchProcessor bool
```

Comment:
This block of fields has grown quite large. I think we should either insert an empty line between each of Otel/Message/Trace/Metric/Log, or convert these field comments to be trailing so the sections are not disrupted:
```suggestion
	EmitterBatchProcessor bool // Enabled by default. Disable only for testing.
```